test/passing/break_cases_all.ml

```diff
@@ -14,70 +13,71 @@ let f x = function
 let f =
   let g = function
     | H
-      when x y <> k ->
-        2
+      when x y <> k -> 2
```

Comment:
Maybe not for this PR, but is the break before `when` intended?

Reply:
fixed